wagtail_transfer/views.py

```diff
@@ -54,6 +56,49 @@ def pages_for_export(request, root_page_id):
     }, json_dumps_params={'indent': 2})
 
 
+@csrf_exempt
+@require_POST
+def objects_for_export(request):
+    """
+    Accepts a POST request with a JSON payload structured as:
+        {
+            'model_label': [list of IDs],
+            'model_label': [list of IDs],
+        }
+    and returns an API response with objects / mappings populated (but ids_for_import empty).
+    """
+    request_data = json.loads(request.body.decode('utf-8'))
+
+    objects = []
+    object_references = set()
+
+    for model_path, ids in request_data.items():
+        model = get_model_for_path(model_path)
```

Comment:
This currently allows exporting any model, and there's no authentiation yet. So this opens up a pretty big security issue. For example:

```shell
curl -X POST "http://localhost:8000/wagtail-transfer/api/objects/" -H 'Content-Type: application/json' -d'
{
    "auth.user": [3]
}
'
```

Returns:

```json
{
  "ids_for_import": [],
  "mappings": [
    [
      "auth.user",
      3,
      "e7f0628c-16a4-11ea-8000-0800278dc04d"
    ]
  ],
  "objects": [
    {
      "model": "auth.user",
      "pk": 3,
      "fields": {
        "password": "pbkdf2_sha256$150000$p7lNCdOnD0Sl$95snmMf0dJNgbSUSqzZVUDk/D4PJOwZVnFq7USi1rbA=",
        "last_login": "2019-12-04T14:42:08.952Z",
        "is_superuser": true,
        "username": "admin",
        "first_name": "Admin",
        "last_name": "User",
        "email": "admin@example.com",
        "is_staff": true,
        "is_active": true,
        "date_joined": "2019-02-17T07:37:53.700Z",
        "groups": "[]",
        "user_permissions": "[]"
      }
    }
  ]
}
``

Reply:
Fair point! Have now added HMAC-based authentication in be8b18da3a2575de545cc20970c8c5d294f6ad2b.